### PR TITLE
moved gas limit above the send button

### LINKED ISSUE #855: show gas up limit above the send button

### PR DESCRIPTION
**Pull request by gyanpgupta**
/kind ui-change
Fixes #855

### CODE REVIEW

**Comment by gyanpgupta** [comment]
@puncsky  updated requested changes please have a look thanks

File: src/shared/wallet/transfer/transfer.tsx [hunk 1]
```diff
@@ -270,6 +277,23 @@ class TransferForm extends React.PureComponent<Props, State> {
...
+        >
+          <div
+            style={{
+              padding: "4px 11px",
+              border: "1px solid #d9d9d9",
+              borderRadius: 4,
+              ...inputStyle
+            }}
+          >
+            {`${toRau(this.state.gasCostLimit.toString(), "Iotx")} IOTX`}
```
Comment (nmhung1210) [inline src/shared/wallet/transfer/transfer.tsx]: It should be `fromRau` here.

File: src/shared/wallet/transfer/transfer.tsx [hunk 2]
```diff
@@ -270,6 +277,23 @@ class TransferForm extends React.PureComponent<Props, State> {
...
+        >
+          <div
+            style={{
+              padding: "4px 11px",
+              border: "1px solid #d9d9d9",
+              borderRadius: 4,
+              ...inputStyle
+            }}
+          >
+            {`${fromRau(this.state.gasCostLimit.toString(), "Iotx")} IOTX`}
```
Comment (nmhung1210) [inline src/shared/wallet/transfer/transfer.tsx]: gasCostLimit = fromRau(`${toRau(gasPrice, "Qev") * gasLimit}`, "IoTx");

**Comment by gyanpgupta** [comment]
gasCostlimit

**Comment by gyanpgupta** [comment]
updatedLimit

**Pull request merged**